src/exporter.h

```diff
@@ -0,0 +1,191 @@
+            << endl;
+    outfile << space1 << "xsi:schemaLocation=\"http://morphml.org/neuroml/"
+                         "schema NeuroML_Level3_v1.7.1.xsd\" "
+            << endl;
+    outfile << space1 << "lengthUnits=\"micrometer\" " << endl;
+
+    outfile << space1 << "<cells>" << endl;
+
+    // In the future, the electrophysiological properties of neurons can be
+    // inserted instead of these prespecified values
+    outfile << space2 << "<cell name=\"exz_lif\"> " << endl;
+    outfile << space3 << "<meta:properties>" << endl;
+    outfile << space4
+            << "<meta:property tag=\"PointNeuronModel\" value=\"yes\"/>"
+            << endl;
+    outfile << space4
+            << "<meta:property tag=\"Class\"    value=\"CbLifNeuron\"/>"
+            << endl;
+    outfile << space4 << "<meta:property tag=\"Cm\"       value=\"5e-10\"/>"
+            << endl;
+    outfile << space4 << "<meta:property tag=\"Rm\"       value=\"1e8\"/>"
+            << endl;
+    outfile << space4 << "<meta:property tag=\"Vthresh\"  value=\"-50e-3\"/>"
+            << endl;
+    outfile << space4 << "<meta:property tag=\"Vresting\" value=\"-60e-3\"/>"
+            << endl;
+    outfile << space4 << "<meta:property tag=\"Vreset\"   value=\"-60e-3\"/>"
+            << endl;
+    outfile << space4 << "<meta:property tag=\"Trefract\" value=\"5e-3\"/>"
+            << endl;
+    outfile << space4 << "<meta:property tag=\"Vinit\"    value=\"-60e-3\"/>"
+            << endl;
+    outfile << space3 << "</meta:properties>" << endl;
+    outfile << space2 << "</cell>" << endl;
+
+    outfile << space2 << "<cell name=\"inh_lif\"> " << endl;
+    outfile << space3 << "<meta:properties>" << endl;
+    outfile << space4
+            << "<meta:property tag=\"PointNeuronModel\" value=\"yes\"/>"
+            << endl;
+    outfile << space4
+            << "<meta:property tag=\"Class\"    value=\"CbLifNeuron\"/>"
+            << endl;
+    outfile << space4 << "<meta:property tag=\"Cm\"       value=\"2e-10\"/>"
+            << endl;
+    outfile << space4 << "<meta:property tag=\"Rm\"       value=\"1e8\"/>"
+            << endl;
+    outfile << space4 << "<meta:property tag=\"Vthresh\"  value=\"-50e-3\"/>"
+            << endl;
+    outfile << space4 << "<meta:property tag=\"Vresting\" value=\"-60e-3\"/>"
+            << endl;
+    outfile << space4 << "<meta:property tag=\"Vreset\"   value=\"-60e-3\"/>"
+            << endl;
+    outfile << space4 << "<meta:property tag=\"Trefract\" value=\"5e-3\"/>"
+            << endl;
+    outfile << space4 << "<meta:property tag=\"Vinit\"    value=\"-60e-3\"/>"
+            << endl;
+    outfile << space3 << "</meta:properties>" << endl;
+    outfile << space2 << "</cell>" << endl;
+    outfile << space1 << "</cells>" << endl;
+
+    // here, the cell populations and connectivity will be
+    // specified and exported
+    for (size_t j = 0; j < VcBackend::kVecLen; j++) {
+      for (size_t i = 0; i < n_vectors; i++) {
```

Comment:
Please see comment on line 38

is it intended that the implementation is missing here?
if yes I would recommend to add a comment `// TODO(roman) ....`